homepage/main.css

```diff
@@ -0,0 +1,16 @@
+:root{
+    --primary-color: #34D399;
+    --secondary-color: ;
```

Comment:
You have not assigned any color to this variable, I think this will result in an error.